server/lib/exporters/__init__.py

```diff
@@ -131,28 +146,26 @@ def append_aggergate_checksums(self):
         """
         aggs = self.manifest["aggregates"]
         for path, chksum in self.state['md5']:
-            uri = "./" + path.replace("data/", "", 1)
+            uri = "./" + path.replace("data/", "")
```

Comment:
what if there's a `data` folder inside `data` folder, or even `data` inside the data inside the data? People did crazier things...

Reply:
Not sure why I dropped that, good catch.